client/src/App.js

```diff
@@ -3,13 +3,16 @@ import NavBarFooter from './components/NavBarFooter.js';
 import GlobalStyles from './GlobalStyles';
 import light from './themes/light.js';
 import { ThemeProvider } from 'emotion-theming';
+import JourneyCard from './components/JourneyCard.js';
+import SpendingCard from './components/SpendingCard.js';
 
 export default function App() {
   return (
     <>
       <ThemeProvider theme={light}>
```

Comment:
do you really need a theme provider, if you import your theme in every component?
I would recommend to use the theme provider, so you can change the theme easily, if you want.